.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test Code
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Also do we want to run this on `master`? Usually I just think blocking PRs if the tests fail is important.